README.md

```diff
@@ -48,6 +48,6 @@ It's that simple. And when it comes time to scale the application, all you have
 
 I know right?
 
-## Contributing
+### Contributing
```

Comment:
I don't know... I'd be careful with adding extra hashtags, as hashtags often appear at the beginning of a program. It will probably be fine here because that's not a program but still.